services/ATOMIC-SWAPS/PROVIDERS/ALLOW.json

```diff
@@ -0,0 +1,20 @@
+[
```

Comment:
This should be part of the lists, not part of this repo. The extension will not make external calls to get this approval list.

We'll allow swap on all sites that are verified in the lists repo.